### PR TITLE
fix: use alpine for lnd-simnet

### PR DESCRIPTION
**Pull request by reliveyy**

### CODE REVIEW

File: images/lnd-simnet/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-RUN apk add --no-cache bash supervisor tor
-COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
-COPY lnd-btc.conf /root/lnd-btc.conf
-COPY lnd-ltc.conf /root/lnd-ltc.conf
-COPY torrc /etc/tor/torrc
-COPY peers.sh /
+FROM alpine:$ALPINE_VERSION
+RUN apk add --no-cache bash expect supervisor tor
+COPY --from=xud-simnet /root/xud-simnet/go/bin/lnd /root/xud-simnet/go/bin/lncli /usr/local/bin/
+COPY entrypoint.sh wallet.exp unlock.exp unlocker.sh wait-file.sh peers.sh /
```
Comment (erkarl) [inline images/lnd-simnet/Dockerfile]: Why did we move `entrypoint.sh`, `wallet.exp`, `unlock.exp`, `unlocker.sh` under lnd-simnet. Isn't it the same for all networks?

File: README.md [hunk 2]
```diff
@@ -5,35 +5,55 @@ A 0-install [xud](https://github.com/ExchangeUnion/xud) environment using [docke
...
-`git submodule update --recursive --remote`
 
-#### Developing a feature
+```bash
+cd $PROJECT_DIR
+git submodule init
+git submodule update --recursive --remote
+```
+
+### Developing a feature
```
Comment (erkarl) [inline README.md]: Thanks for updating the README. Do these steps work in current master?

Comment (reliveyy) [reply README.md]: Not yet

File: images/lnd-simnet/Dockerfile (hunk 1 above)
Comment (reliveyy) [reply images/lnd-simnet/Dockerfile]: I'll remove unused files later. Copied files just for quick test.

File: README.md (hunk 2 above)
Comment (erkarl) [reply README.md]: In that case I think we should omit these changes from this PR and add them when the new functionality is working.

Comment (reliveyy) [reply README.md]: If I cannot finish the rest of this PR today, I will split it.

File: README.md [hunk 3]
```diff
@@ -5,35 +5,76 @@ A 0-install [xud](https://github.com/ExchangeUnion/xud) environment using [docke
...
+$PROJECT_DIR/tools/build <image_name>
+```
+
+Test locally
+
+```bash
+$PROJECT_DIR/xud.sh -b your-feature-branch
+```
+
+Push images of your-feature-branch and let others test without build images by themselves
```
Comment (erkarl) [inline README.md]:
```suggestion
To let others test without building the images by themselves push the images of your-feature-branch.
```

File: README.md [hunk 4]
```diff
@@ -5,35 +5,76 @@ A 0-install [xud](https://github.com/ExchangeUnion/xud) environment using [docke
...
+git push origin your-feature-branch
+```
+
+Ask other people to run your-feature-branch on their machine
+
+```bash
+./xud.sh -b your-feature-branch
+```
+
+Or they need to build images of your-feature-branch test like you do it locally.
```
Comment (erkarl) [inline README.md]: I think this line is not necessary as we indirectly already mention it above.

File: README.md [hunk 5]
```diff
@@ -5,35 +5,76 @@ A 0-install [xud](https://github.com/ExchangeUnion/xud) environment using [docke
...
+
+### Test on cloud
+
+We know running simnet, testnet and mainnet simultaneously on personal computer is nearly impossible. So we provide a convenient `tools/test` to let your test your-feature-branch on cloud (We only support Google Cloud for now)
```
Comment (erkarl) [inline README.md]:
```suggestion
Running simnet, testnet and mainnet simultaneously is resource heavy. We provide a convenient `tools/test` script to test your-feature-branch in the cloud. Currently, only Google Cloud is supported.
```

File: README.md [hunk 6]
```diff
@@ -5,35 +5,76 @@ A 0-install [xud](https://github.com/ExchangeUnion/xud) environment using [docke
...
+
+1. Download `google-cloud-sdk` on your machine.
+2. Running `gcloud init` to login to your Google account and choose the project and region
```
Comment (erkarl) [inline README.md]:
```suggestion
2. Run `gcloud init` to login to your Google account and choose the project and region
```

File: images/lnd-simnet/Dockerfile (hunk 1 above)
Comment (reliveyy) [reply images/lnd-simnet/Dockerfile]: I tried to remove unlocker.sh for simnet but failed. I think it's the time to merge this issue.

**Pull request merged**